pkg/apis/kops/v1alpha1/zz_generated.conversion.go

```diff
@@ -2662,14 +2662,10 @@ func autoConvert_kops_LoadBalancerAccessSpec_To_v1alpha1_LoadBalancerAccessSpec(
 	out.Type = LoadBalancerType(in.Type)
 	out.IdleTimeoutSeconds = in.IdleTimeoutSeconds
 	out.AdditionalSecurityGroups = in.AdditionalSecurityGroups
+	// WARNING: in.UseApiInternal requires manual conversion: does not exist in peer-type
```

Comment:
Ah - this means you need to add the new field to v1alpha1 also...

Reply:
ah ok, I wasn't sure if v1alpha1 was an API that should not change and what I was supposed to do was add the field only to v1alpha2 and have a conversion function.  Will add it here as well.